juanevillam/xtrovert
# Add Login Form with Validation and Reusable Components

## Overview  
This PR adds a login form with validation using `zod` and introduces reusable UI components. It also configures the primary and success colors for consistent styling across the application.

---

## Key Changes  

### **Features**  
- Created a login form with `zod` validation (`loginSchema`).  
- Developed reusable components:  
  - `AuthCard`  
  - `AuthTextInput`  
  - `AuthLink`  
  - `Button`  

### **Enhancements**  
- Updated and implemented primary and success colors in the theme for consistent styling.  

---

## Reasons for Changes  
These changes enable user authentication via a login form with proper validation, enhancing security and user experience. Introducing reusable components promotes consistency and maintainability in the UI. Standardizing colors ensures a cohesive look and feel throughout the application.

---

## How to Test  
1. Run the application and navigate to the login page.  
2. Verify that the login form displays correctly.  
3. Enter valid and invalid inputs to test form validation and error handling.  
4. Ensure that the reusable components function as expected in different parts of the application.  
5. Check that the primary and success colors are applied consistently across the UI.
